src/triage/component/timechop/plotting.py

```diff
@@ -110,4 +112,5 @@ def visualize_chops(chopper, show_as_of_times=True, show_boundaries=True):
     ax[0].set_title("Timechop: Temporal cross-validation blocks")
     fig.subplots_adjust(hspace=0)
     plt.setp([a.get_xticklabels() for a in fig.axes[:-1]], visible=False)
+    plt.savefig('timechop.png')
```

Comment:
Isn't this going to break (and never call show) if whatever is running doesn't have write access to whatever directory this ends up being? Furthermore, if this breaks in a notebook setting (when the user is just trying to plot it to the screen), it will break their use case for saving functionality that they don't even care about.

I think there's a lot that we can do to make this more robust (integrate it with the ProjectStorage via the CLI, etc) but the bare minimum for this pull request I think should be to just plot to an optional target instead of hardcoding a filesystem path. We do this in the audition plotting module for an example.

https://github.com/dssg/triage/blob/master/src/triage/component/audition/plotting.py#L217-L218

Just have the save target be an optional argument, and conditionally save it like this:

```
if path_to_save:
    plt.savefig(path_to_save)
```

And we can make the implementation more complete in a future PR